src/ui/public/kuery/ast/ast.js

```diff
@@ -1,21 +1,25 @@
 import grammar from 'raw-loader!./kuery.peg';
+import simpleGrammar from 'raw-loader!./simple_kuery.peg';
 import PEG from 'pegjs';
 import _ from 'lodash';
 import { nodeTypes } from '../node_types/index';
 
 const kueryParser = PEG.buildParser(grammar);
+const simpleKueryParser = PEG.buildParser(simpleGrammar);
 
-export function fromKueryExpression(expression, parseOptions = {}) {
+export function fromKueryExpression(expression, parseOptions = {}, useSimpleParser = false) {
```

Comment:
Similar to the comment above, I think a separate `fromSimpleKueryExpression` function would be more clear than a boolean flag here.